framework/helpers/BaseStringHelper.php

```diff
@@ -45,6 +45,23 @@ public static function byteSubstr($string, $start, $length = null)
     {
         return mb_substr($string, $start, $length === null ? mb_strlen($string, '8bit') : $length, '8bit');
     }
+    
+    /**
+     * Returns given string with first character uppercased
+     * @param string $string
+     * @param string $encoding
+     * @return string
+     */
+    public static function ucfirst($string, $encoding = null)
+    {
+        if (!$encoding) {
+            $encoding = \mb_internal_encoding();
```

Comment:
Why leading backslash is used here? It's also a function from "mb" group, not a class name.